There seems to be a unit test failure:
```
--- FAIL: TestListContainers (0.00s)
	assertions.go:203: 
                        
	Error Trace:	docker_container_test.go:96
		
	Error:		Not equal: []*runtime.Container{(*runtime.Container)(0xc4201694a0), (*runtime.Container)(0xc4201693e0), (*runtime.Container)(0xc420169320)} (expected)
			        != []*runtime.Container{(*runtime.Container)(0xc420169560), (*runtime.Container)(0xc420169620), (*runtime.Container)(0xc4201696e0)} (actual)
			
			Diff:
			--- Expected
			+++ Actual
			@@ -1,5 +1,5 @@
			 ([]*runtime.Container) (len=3 cap=4) {
			- (*runtime.Container)(0xc4201694a0)(&Container{Id:fdfc05820c316461,PodSandboxId:sandboxid2,Metadata:&ContainerMetadata{Name:sidecar2,Attempt:2,},Image:&ImageSpec{Image:logger:v2,},ImageRef:,State:CONTAINER_RUNNING,CreatedAt:0,Labels:map[string]string{abc.xyz: label2,},Annotations:map[string]string{foo.bar.baz: annotation2,},}),
			- (*runtime.Container)(0xc4201693e0)(&Container{Id:bd837aa611586636,PodSandboxId:sandboxid1,Metadata:&ContainerMetadata{Name:sidecar1,Attempt:1,},Image:&ImageSpec{Image:logger:v1,},ImageRef:,State:CONTAINER_RUNNING,CreatedAt:0,Labels:map[string]string{abc.xyz: label1,},Annotations:map[string]string{foo.bar.baz: annotation1,},}),
			- (*runtime.Container)(0xc420169320)(&Container{Id:4c1e1e211ef4ba57,PodSandboxId:sandboxid0,Metadata:&ContainerMetadata{Name:sidecar0,Attempt:0,},Image:&ImageSpec{Image:logger:v0,},ImageRef:,State:CONTAINER_RUNNING,CreatedAt:0,Labels:map[string]string{abc.xyz: label0,},Annotations:map[string]string{foo.bar.baz: annotation0,},})
			+ (*runtime.Container)(0xc420169560)(&Container{Id:fdfc05820c316461,PodSandboxId:sandboxid2,Metadata:&ContainerMetadata{Name:sidecar2,Attempt:2,},Image:&ImageSpec{Image:logger:v2,},ImageRef:,State:CONTAINER_RUNNING,CreatedAt:-6795364578871345152,Labels:map[string]string{abc.xyz: label2,},Annotations:map[string]string{foo.bar.baz: annotation2,},}),
			+ (*runtime.Container)(0xc420169620)(&Container{Id:bd837aa611586636,PodSandboxId:sandboxid1,Metadata:&ContainerMetadata{Name:sidecar1,Attempt:1,},Image:&ImageSpec{Image:logger:v1,},ImageRef:,State:CONTAINER_RUNNING,CreatedAt:-6795364578871345152,Labels:map[string]string{abc.xyz: label1,},Annotations:map[string]string{foo.bar.baz: annotation1,},}),
			+ (*runtime.Container)(0xc4201696e0)(&Container{Id:4c1e1e211ef4ba57,PodSandboxId:sandboxid0,Metadata:&ContainerMetadata{Name:sidecar0,Attempt:0,},Image:&ImageSpec{Image:logger:v0,},ImageRef:,State:CONTAINER_RUNNING,CreatedAt:-6795364578871345152,Labels:map[string]string{abc.xyz: label0,},Annotations:map[string]string{foo.bar.baz: annotation0,},})
			 }
		
E0308 18:18:55.712999       6 helpers.go:296] Failed to remove the conflicting container "containerid": container not stopped
E0308 18:18:55.713130       6 helpers.go:296] Failed to remove the conflicting container "containerid": random error
E0308 18:18:55.713182       6 helpers.go:296] Failed to remove the conflicting container "containerid": Error response from daemon: No such container: containerid
--- FAIL: TestListSandboxes (0.00s)
	assertions.go:203: 
                        
	Error Trace:	docker_sandbox_test.go:87
		
	Error:		Not equal: []*runtime.PodSandbox{(*runtime.PodSandbox)(0xc4201d5dc0), (*runtime.PodSandbox)(0xc4201d5940), (*runtime.PodSandbox)(0xc4201d56c0)} (expected)
			        != []*runtime.PodSandbox{(*runtime.PodSandbox)(0xc4201d5f00), (*runtime.PodSandbox)(0xc4201280c0), (*runtime.PodSandbox)(0xc420128300)} (actual)
			
			Diff:
			--- Expected
			+++ Actual
			@@ -1,5 +1,5 @@
			 ([]*runtime.PodSandbox) (len=3 cap=4) {
			- (*runtime.PodSandbox)(0xc4201d5dc0)(&PodSandbox{Id:18601f7115bbb09,Metadata:&PodSandboxMetadata{Name:foo2,Uid:2,Namespace:bar2,Attempt:0,},State:SANDBOX_READY,CreatedAt:0,Labels:map[string]string{label: foo2,},Annotations:map[string]string{annotation: bar2,},}),
			- (*runtime.PodSandbox)(0xc4201d5940)(&PodSandbox{Id:89ff0a41f3c737ca,Metadata:&PodSandboxMetadata{Name:foo1,Uid:1,Namespace:bar1,Attempt:0,},State:SANDBOX_READY,CreatedAt:0,Labels:map[string]string{label: foo1,},Annotations:map[string]string{annotation: bar1,},}),
			- (*runtime.PodSandbox)(0xc4201d56c0)(&PodSandbox{Id:9c5f40c0a5ad48bf,Metadata:&PodSandboxMetadata{Name:foo0,Uid:0,Namespace:bar0,Attempt:0,},State:SANDBOX_READY,CreatedAt:0,Labels:map[string]string{label: foo0,},Annotations:map[string]string{annotation: bar0,},})
			+ (*runtime.PodSandbox)(0xc4201d5f00)(&PodSandbox{Id:18601f7115bbb09,Metadata:&PodSandboxMetadata{Name:foo2,Uid:2,Namespace:bar2,Attempt:0,},State:SANDBOX_READY,CreatedAt:-6795364578871345152,Labels:map[string]string{label: foo2,},Annotations:map[string]string{annotation: bar2,},}),
			+ (*runtime.PodSandbox)(0xc4201280c0)(&PodSandbox{Id:89ff0a41f3c737ca,Metadata:&PodSandboxMetadata{Name:foo1,Uid:1,Namespace:bar1,Attempt:0,},State:SANDBOX_READY,CreatedAt:-6795364578871345152,Labels:map[string]string{label: foo1,},Annotations:map[string]string{annotation: bar1,},}),
			+ (*runtime.PodSandbox)(0xc420128300)(&PodSandbox{Id:9c5f40c0a5ad48bf,Metadata:&PodSandboxMetadata{Name:foo0,Uid:0,Namespace:bar0,Attempt:0,},State:SANDBOX_READY,CreatedAt:-6795364578871345152,Labels:map[string]string{label: foo0,},Annotations:map[string]string{annotation: bar0,},})
			 }
		
FAIL

```

--- a/pkg/kubelet/dockershim/docker_container_test.go
+++ b/pkg/kubelet/dockershim/docker_container_test.go
@@ -47,7 +47,7 @@ func makeContainerConfig(sConfig *runtimeapi.PodSandboxConfig, name, image strin
 // TestListContainers creates several containers and then list them to check
 // whether the correct metadatas, states, and labels are returned.
 func TestListContainers(t *testing.T) {
-	ds, _, _ := newTestDockerService()
+	ds, _, fakeClock := newTestDockerService()
 	podName, namespace := "foo", "bar"
 	containerName, image := "sidecar", "logger"
 
@@ -66,7 +66,7 @@ func TestListContainers(t *testing.T) {
 
 	expected := []*runtimeapi.Container{}
 	state := runtimeapi.ContainerState_CONTAINER_RUNNING
-	var createdAt int64 = 0
+	var createdAt int64 = fakeClock.Now().UnixNano()
 	for i := range configs {
 		// We don't care about the sandbox id; pass a bogus one.
 		sandboxID := fmt.Sprintf("sandboxid%d", i)

--- a/pkg/kubelet/dockershim/docker_sandbox_test.go
+++ b/pkg/kubelet/dockershim/docker_sandbox_test.go
@@ -52,7 +52,7 @@ func makeSandboxConfigWithLabelsAndAnnotations(name, namespace, uid string, atte
 // TestListSandboxes creates several sandboxes and then list them to check
 // whether the correct metadatas, states, and labels are returned.
 func TestListSandboxes(t *testing.T) {
-	ds, _, _ := newTestDockerService()
+	ds, _, fakeClock := newTestDockerService()
 	name, namespace := "foo", "bar"
 	configs := []*runtimeapi.PodSandboxConfig{}
 	for i := 0; i < 3; i++ {
@@ -66,7 +66,7 @@ func TestListSandboxes(t *testing.T) {
 
 	expected := []*runtimeapi.PodSandbox{}
 	state := runtimeapi.PodSandboxState_SANDBOX_READY
-	var createdAt int64 = 0
+	var createdAt int64 = fakeClock.Now().UnixNano()
 	for i := range configs {
 		id, err := ds.RunPodSandbox(configs[i])
 		assert.NoError(t, err)

--- a/pkg/kubelet/dockertools/convert.go
+++ b/pkg/kubelet/dockertools/convert.go
@@ -28,6 +28,7 @@ import (
 // (kubecontainer) types.
 const (
 	statusRunningPrefix = "Up"
+	statusCreatedPrefix = "Created"
 	statusExitedPrefix  = "Exited"
 )
 

--- a/pkg/kubelet/dockertools/fake_docker_client.go
+++ b/pkg/kubelet/dockertools/fake_docker_client.go
@@ -516,12 +516,13 @@ func (f *FakeDockerClient) CreateContainer(c dockertypes.ContainerCreateConfig) 
 	name := "/" + c.Name
 	id := GetFakeContainerID(name)
 	f.appendContainerTrace("Created", id)
+	timestamp := f.Clock.Now()
 	// The newest container should be in front, because we assume so in GetPodStatus()
 	f.RunningContainerList = append([]dockertypes.Container{
-		{ID: id, Names: []string{name}, Image: c.Config.Image, Labels: c.Config.Labels},
+		{ID: id, Names: []string{name}, Image: c.Config.Image, Created: timestamp.Unix(), State: statusCreatedPrefix, Labels: c.Config.Labels},
 	}, f.RunningContainerList...)
 	f.ContainerMap[id] = convertFakeContainer(&FakeContainer{
-		ID: id, Name: name, Config: c.Config, HostConfig: c.HostConfig, CreatedAt: f.Clock.Now()})
+		ID: id, Name: name, Config: c.Config, HostConfig: c.HostConfig, CreatedAt: timestamp})
 
 	f.normalSleep(100, 25, 25)
 
@@ -539,12 +540,13 @@ func (f *FakeDockerClient) StartContainer(id string) error {
 	}
 	f.appendContainerTrace("Started", id)
 	container, ok := f.ContainerMap[id]
+	timestamp := f.Clock.Now()
 	if !ok {
-		container = convertFakeContainer(&FakeContainer{ID: id, Name: id})
+		container = convertFakeContainer(&FakeContainer{ID: id, Name: id, CreatedAt: timestamp})
 	}
 	container.State.Running = true
 	container.State.Pid = os.Getpid()
-	container.State.StartedAt = dockerTimestampToString(f.Clock.Now())
+	container.State.StartedAt = dockerTimestampToString(timestamp)
 	container.NetworkSettings.IPAddress = "2.3.4.5"
 	f.ContainerMap[id] = container
 	f.updateContainerStatus(id, statusRunningPrefix)

